contracts/0.4.24/Lido.sol

```diff
@@ -440,28 +690,43 @@ contract Lido is ILido, IsContract, StETH, AragonApp {
     * @return amount of StETH shares generated
     */
     function _submit(address _referral) internal whenNotStopped returns (uint256) {
```

Comment:
Idea for future may be postponed: save us 2100 gas - remove the modifier and add the code that actually set staking limit to the stop() method.

Reply:
You mean that `stop()` also effectively should call `pauseStaking()`? 
Interesting, but maybe non-expected outside :thinking: